lib/writer/block-write-stream.js

```diff
@@ -95,6 +97,27 @@ class BlockWriteStream extends stream.Writable {
     this.open()
   }
 
+  /**
+   * @summary Determine whether an error is considered a
+   * transient occurrence, and the operation should be retried
+   * Errors considered potentially temporary are:
+   *   - Mac OS: ENXIO, EBUSY
+   *   - Windows: ENOENT, UNKNOWN
+   *   - Linux: EIO
+   * @private
+   * @param {Error} error - Error
+   * @returns {Boolean}
+   * @example
+   * BlockWriteStream.isTransientError(error)
+   */
+  static isTransientError (error) {
```

Comment:
Can we somehow re-use this function from the GUI side? Looks like we're basically re-copying this whole check.

Reply:
Done